index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newton Portfolio Project</title>
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <main>
+    <nav>
+      <div class="desktop-menu">
+        <ul>
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <div class="mobile-menu-icon">
+        <img src="./images/Union.svg" alt="">
+      </div>
+    </nav>
+    <section class="home">
+      <div class="introduction">
+        <h2>Hey there.</h2>
+        <h2>I'm Newton.</h2>
+        <h2>I'm a software developer</h2>
+      </div>
+      <div class="content">
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div>
```

Comment:
Your design
![image](https://github.com/NewtonOmbese/Newton-Portfolio/assets/78251927/032af47a-322d-4c21-89ef-a60df24d0ee2)

Figma
![image](https://github.com/NewtonOmbese/Newton-Portfolio/assets/78251927/e73f0d64-5d45-4e9e-b3c0-d0732965b4af)

Kindly ensure that the section uses the `font` and `colors` defined for the template.

* If it's pretty and works it will be a wonderful website.

[OPTIONAL] - I would suggest that we remove the generic `div` enclosing the `p` tag. 

It does not have any special purpose there. That way will be [semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct. 👍

Reply:
@abdulhamiid this is how my code looks, is the paragraph ok or should I do some changes since I followed the figma typography of the paragraph and implemented them, having a font weight of 400, a font size of 20px and a line height of 28px
![Screenshot from 2023-06-01 08-40-54](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/4510bc69-ae49-426a-bbfc-1c476ae29e10)